### ISSUE TITLE
Emojis

### ISSUE THREAD

**Issue by BrunoMiguens** [traduora/traduora#26]
**Describe the bug**
Using a translation file with emojis on their translations, the web app throw an error and don't import any translation.

**To Reproduce**
1. Use a translation file (I've used a `.strings` file)
2. Import the file 
3. An error will be thrown

**Expected behavior**
The ideal solution is to import the translations and terms with the emojis, however, would be great if the web app could import at least the rest of the strings without throwing an error.

**Environment (please complete the following information):**
 - Device: Desktop
 - OS: Mac OS
 - Browser safari
 - Version 0.9.0

**Comment by anthonynsimon** [comment]
@BrunoMiguens Hey this seems to be related to #22 which is already merged but not yet released on a new version. Could you give it a try by using the code on master and let me know?

**Comment by BrunoMiguens** [comment]
@anthonynsimon Still happening...

```
POST /api/v1/projects/e965f5e4-32cd-4d8a-8c35-45599eab39fe/imports?format=strings&locale=pt_AO HTTP/1.1 500 83 - 176.575 ms
traduora_1  | { QueryFailedError: ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: '\xE2\x9A\xA0\xEF\xB8\x8F...' for column 'value' at row 1
traduora_1  |     at new QueryFailedError (/opt/traduora/node_modules/typeorm/error/QueryFailedError.js:28:28)
traduora_1  |     at Query.<anonymous> (/opt/traduora/node_modules/typeorm/driver/mysql/MysqlQueryRunner.js:227:45)
traduora_1  |     at Query.<anonymous> (/opt/traduora/node_modules/mysql/lib/Connection.js:502:10)
traduora_1  |     at Query._callback (/opt/traduora/node_modules/mysql/lib/Connection.js:468:16)
traduora_1  |     at Query.Sequence.end (/opt/traduora/node_modules/mysql/lib/protocol/sequences/Sequence.js:83:24)
traduora_1  |     at Query.ErrorPacket (/opt/traduora/node_modules/mysql/lib/protocol/sequences/Query.js:90:8)
traduora_1  |     at Protocol._parsePacket (/opt/traduora/node_modules/mysql/lib/protocol/Protocol.js:278:23)
traduora_1  |     at Parser.write (/opt/traduora/node_modules/mysql/lib/protocol/Parser.js:76:12)
traduora_1  |     at Protocol.write (/opt/traduora/node_modules/mysql/lib/protocol/Protocol.js:38:16)
traduora_1  |     at Socket.<anonymous> (/opt/traduora/node_modules/mysql/lib/Connection.js:91:28)
traduora_1  |   message:
traduora_1  |    'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD: Incorrect string value: \'\\xE2\\x9A\\xA0\\xEF\\xB8\\x8F...\' for column \'value\' at row 1',
traduora_1  |   code: 'ER_TRUNCATED_WRONG_VALUE_FOR_FIELD',
traduora_1  |   errno: 1366,
traduora_1  |   sqlMessage:
traduora_1  |    'Incorrect string value: \'\\xE2\\x9A\\xA0\\xEF\\xB8\\x8F...\' for column \'value\' at row 1',
traduora_1  |   sqlState: 'HY000',
traduora_1  |   index: 0,
traduora_1  |   sql:
traduora_1  |    'INSERT INTO `translation`(`termId`, `projectLocaleId`, `value`, `dateCreated`, `dateModified`) VALUES (\'b32c7465-f4c2-415f-a7e3-c77cc6a0f0fe\', \'9a98bcad-e166-4413-b584-62985926f058\', \'⚠️ Sample\', DEFAULT, DEFAULT)',
traduora_1  |   name: 'QueryFailedError',
traduora_1  |   query:
traduora_1  |    'INSERT INTO `translation`(`termId`, `projectLocaleId`, `value`, `dateCreated`, `dateModified`) VALUES (?, ?, ?, DEFAULT, DEFAULT)',
traduora_1  |   parameters:
traduora_1  |    [ 'b32c7465-f4c2-415f-a7e3-c77cc6a0f0fe',
traduora_1  |      '9a98bcad-e166-4413-b584-62985926f058',
traduora_1  |      '⚠️ Sample' ] }
```

**Comment by anthonynsimon** [comment]
@BrunoMiguens Are you building from source on master or using the docker image?

**Comment by anthonynsimon** [comment]
I just released version `0.10.0` which includes this fix. Just be sure to pull the latest docker image to try it out:

```
docker pull traduora/traduora
```
